packages/generator-widget/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@mendix/generator-widget",
-  "version": "10.7.2",
+  "version": "10.7.3",
```

Comment:
Should we follow the current version of SP? Or indeed treat this release as a patch?

Reply:
I think we should treat it as a patch, there are no specific SP changes required for the included patches.